source/dotnet/Library/AdaptiveCards.Templating.CSharp.WinRT/AdaptiveCards.Template.csproj

```diff
@@ -9,6 +9,9 @@
 
   <PropertyGroup>   
     <CsWinRTComponent>true</CsWinRTComponent>
+    <SignAssembly>True</SignAssembly>   
+    <AssemblyOriginatorKeyFile>../../../../35MSSharedLib1024.snk</AssemblyOriginatorKeyFile>   
```

Comment:
nit: can the dir containing the key be made into a variable instead (even if it's still `../../../../`)? This technically points outside the Git repo and assumes things about the build setup

Reply:
updated it. let me know if this is what you wanted it.